client/css/components/fragments/Header.css

```diff
@@ -0,0 +1,20 @@
+#Header {
+  z-index: 9;
+  height: 56px;
+}
+
+#Header #logo h1 {
```

Comment:
Could we stay consistent in how we name elements? Header starts with upper case character, logo with lowercase character